prime-dotnet-webapi/Controllers/OrganizationsController.cs

```diff
@@ -487,6 +492,22 @@ public async Task<ActionResult> AcceptOrganizationAgreement(int organizationId,
                 await _organizationService.FinalizeTransferAsync(organizationId);
             }
 
+
+            if (!organizationAgreementGuid.HasValue)
+            {
+                var filename = "Organization-Agreement.pdf";
+                // get the agreement
+                var agreement = await _organizationAgreementService.GetOrgAgreementAsync(organizationId, agreementId, true);
+                // store it into document manager
+                var documentGuid = await _documentManagerClient.SendFileAsync(new System.IO.MemoryStream(Convert.FromBase64String(agreement.AgreementContent)), filename, DestinationFolders.SignedOrgAgreements);
```

Comment:
Wouldn't the latest signed org agreement for **any** organization overwrite the previous files since the filename is identical and the folder is also constant?  I am still trying to confirm as I am not finding the files where I expected them ... where should they be saving to on the file system?

Reply:
In the document manager docker console (my local environment), I can see the filename has been encoded. Therefore, the files with the same name will not be replaced or overwritten. 
![image](https://user-images.githubusercontent.com/108699279/202756807-7c5d9c9e-62a8-4b89-a657-d85c2ee1c878.png)